### PR TITLE
Add documentation how to install fastly-exporter via Helm

### PR DESCRIPTION
**Pull request by arslanbekov**
Added documentation on how to install the exporter via helm chart.
https://github.com/prometheus-community/helm-charts/pull/2278

Signed-off-by: Denis Arslanbekov <arslanbekov@gmail.com>

### CODE REVIEW

**Comment by arslanbekov** [comment]
@peterbourgon hey, can review pls?

**Comment by peterbourgon** [comment]
I think this belongs under Getting rather than Using. Something symmetric to the Docker instructions feels more appropriate.

**Comment by arslanbekov** [comment]
@peterbourgon that's right, thanks. Fixed

File: README.md [hunk 1]
```diff
@@ -30,6 +30,23 @@ docker pull ghcr.io/fastly/fastly-exporter:latest
...
+```sh
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+```
+
+And install:
+
+```sh
+helm upgrade --install fastly-exporter prometheus-fastly-exporter --namespace monitoring --set token="fastly_api_token"
+```
+
```
Comment (peterbourgon) [inline README.md:49]: ### Helm

 ```sh
helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm upgrade --install fastly-exporter prometheus-fastly-exporter --namespace monitoring --set token="fastly_api_token"
```

_edit_: oh

**Pull request merged**